PointerStar/Client/ViewModels/RoomViewModel.cs

```diff
@@ -44,9 +46,12 @@ public bool IsTeamMember
 
     [ObservableProperty]
     private Guid? _selectedRoleId;
-    
+
     public string? RoomId { get; set; }
 
+    public string CopyButtonText { get; set; } = "Copy Invitation Link ";
+    public string CopyButtonIcon { get; set; } = "fa fa-copy";
```

Comment:
So when you move properties from the view to the view model, the "magic" that blazor uses to updated when a property changes is lost. So for the view to update properly when the property changes you will need to make these properties raise the `INotifyPropertyChanged.PropertyChanged` event. This is easiest by using the `[ObservableProperty]` attribute (examples above).

Reply:
ohhh ok. Thank you, this worked. I was surprised it was working before without me notifying it :-)